life_expectancy/loaders.py

```diff
@@ -0,0 +1,16 @@
+"""Save and load functions
```

Comment:
the file name is ambiguous. I would only expect to find loading functions here. Typically here I'd call this persistance.py or serialization.py (which is still a bit linked to loading) or less imaginatively save_load.py

Reply:
Sorry @fddelgadoN, it seems like I suggested a bad name..

I wouldn't call it serialization though. When I read serialization I think of translating an object to a state that can be stored, not the storing itself.

I think `persistence` is fine and I've seen other projects use it. Other suggestions is to use the `io` suffix. For example:

- `csv_io`
- `data_io`
- `dataframe_io`

Finally, you can always use `data_access`  which I find it very on the nose.